Azure/LogicAppsUX
# feat(vscode): Move extension tab to resources tab

### Main code changes
- Add LogicAppResolver which will resolve the logicapps resources and will create the tree.
- Update architecture to rely on resources group tree view.
- Move title ribbon buttons to the workspace title tab (Resources team currently doesn't have a more scalable approach)
- Update register commands to unwrap commands 
- Remove SlotTreeItemBase class and move everything to SlotTreeItem
- Remove ProductionSlotTreeItem class and move main logic of remote logic apps to LogicAppResourceTree
- Update node picker on commands
- Add icon in overview page.
- Add azure resources as extension dependency.
- Remove azure svg.


### Tested scenarios
- Open logic apps remote tree, this implie.
- Open designer and overview pages.
- Check all right click context menu commands in new remote tree structure show up.
- Deploying local to remote.

### Implementation

#### Logic apps tree in resources tab
<img width="540" alt="image" src="https://user-images.githubusercontent.com/102700317/234646788-d482177a-26a5-4790-9a40-0e02cc8d8186.png">

#### Ribbon buttons moved to workspace tab
<img width="560" alt="image" src="https://user-images.githubusercontent.com/102700317/234647074-433f9551-8093-44a4-ae6b-3429ef06a7d2.png">

Closes #1835 #1837 #1830
